### ISSUE TITLE
anyElement/@minOccurs and anyElement/@maxOccurs ignored

### ISSUE THREAD

**Issue by sydb** [TEIC/Stylesheets#297]
Title kinda says it all. But to be clear, I put
~~~~~xml
  <anyElement require="http://www.example.org/useme"
              minOccurs="1" maxOccurs="unbounded"/>
~~~~~
in an ODD, and what came out was just
~~~~~xml
  <rng:ref name="anyElement-SPECNAME"/>
~~~~~
i.e., the occurence wrapper element `<rng:oneOrMore>` was missing.

**Comment by sydb** [comment]
Note: this is a blocker of ticket https://github.com/TEIC/TEI/issues/1716.

**Comment by sydb** [comment]
The error looks to be in teiodds.xsl ~331, the template that matches "anyElement" in "tangle" mode. It just output a single `<rng:ref>`. I’ve changed it to wrap that `<rng:ref>` in tei:generateIndicators(@minOccurs,@maxOccurs). I will check it in as soon as my local build is successful or my boarding group is called, whichever comes first.  :-)

**Comment by sydb** [comment]
Pushed fix at be2a2e8. I think it worked. However, my commit msg is messed up. Is there a way to edit my commit message after the fact?
This ticket should be closed as soon as Mr. Jenkins is happy and someone tests result.
